artemis-server/src/main/java/org/apache/activemq/artemis/core/remoting/impl/netty/NettyAcceptor.java

```diff
@@ -594,21 +596,13 @@ public synchronized SslHandler getSslHandler(ByteBufAllocator alloc, String peer
    private SSLEngine loadJdkSslEngine(String peerHost, int peerPort) throws Exception {
```

Comment:
How come there aren't corresponding changes in `loadOpenSslEngine`?

Reply:
Because they don't produce a SSLContext but a SslContext